lib/trino-orc/src/main/java/io/trino/orc/metadata/statistics/StringStatisticsBuilder.java

```diff
@@ -170,4 +195,73 @@ private Slice dropStringMinMaxIfNecessary(Slice minOrMax)
         }
         return Slices.copyOf(minOrMax);
     }
+
+    static class StringCompactor
```

Comment:
I have two IDE warnings on this line. https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#additional-ide-configuration

Reply:
I have it properly configured. Just didn't notice them. thanks